cvat/apps/engine/templates/engine/annotation.html

```diff
@@ -330,21 +330,21 @@
 
     <div id="annotationMenu" class="hidden regular">
         <center style="float:left; width: 28%; height: 100%;" id="engineMenuButtons">
-            <div class="dropdown">
-                <button id="downloadAnnotationButton" class="menuButton semiBold h2"> Dump Annotation </button>
-                <ul id="downloadDropdownMenu" class="dropdown-content hidden"></ul>
-            </div>
-            <div class="dropdown">
-                <button id="uploadAnnotationButton" class="menuButton semiBold h2"> Upload Annotation </button>
-                <ul id="uploadDropdownMenu" class="dropdown-content hidden"></ul>
-            </div>
+            <select id="downloadAnnotationButton" class="menuButton semiBold h2" style="text-align-last: center;">
+                <option selected disabled> Dump Annotation </option>
+            </select>
+
+            <select id="uploadAnnotationButton" class="menuButton semiBold h2" style="text-align-last: center;">
+                    <option selected disabled> Upload Annotation </option>
+            </select>
+
             <button id="removeAnnotationButton" class="menuButton semiBold h2"> Remove Annotation </button>
             <button id="settingsButton" class="menuButton semiBold h2"> Settings </button>
             <button id="fullScreenButton" class="menuButton semiBold h2"> Fullscreen Player </button>
             <button id="switchAAMButton" class="menuButton semiBold h2"> Switch AAM </button>
             <button id="helpButton" class="menuButton semiBold h2"> Help </button>
             <button id="saveButton" class="menuButton semiBold h2"> Save Work </button>
-            <input type="file" id="annotationFileSelector" style="display: none"/>
+            <input type="file" id="annotationFileSelector" accept="text/xml,application/zip,application/json" style="display: none"/>
```

Comment:
Can we have an annotation format in `txt` format in the future? I like the idea to restrict the selection but really don't like that it is "hard coded".

Reply:
@nmanovic 
Browse window had been restricted before (at least in the dashboard). 
I just expanded it and did it consistent. Probably we can expand it dynamically.

Reply:
Done